test/integration/test-custom-filename.js

```diff
@@ -20,25 +20,26 @@ var server = http.createServer(function(req, res) {
 
   var form = new IncomingForm({uploadDir: common.dir.tmp});
 
-  form.parse(req);
-
-  form
-    .on('file', function(name, file) {
-      assert.strictEqual(name, 'my_file');
-      assert.strictEqual(file.name, options.filename);
-      assert.strictEqual(file.type, options.contentType);
-    })
-    .on('end', function() {
+  form.parse(req, function (err, fields, files) {
+      assert("my_file1" in files);
```

Comment:
Don't want to nitpick, but do you mind to use single quotes as rest of the code does?

And thanks a lot for the driving this thing forward.